simulators/ethereum/consensus/go.sum

```diff
@@ -0,0 +1,582 @@
+bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898/go.mod h1:Xbm+BRKSBEpa4q4hTSxohYNQpsxXPbPry4JJWOB3LB8=
```

Comment:
Damn, this was big. Is all of it needed?

Reply:
Yeah, it's kind of an annoying issue at the moment. It happens because the hivesim package
imports internal/libhive *in its unit tests*, and Go somehow decides to include all the
dependency checksums of all hive dependencies into this file. It happened with all other
simulations as well. I guess it's best to just not look at go.sum.